packages/gatsby-cli/src/structured-errors/construct-error.ts

```diff
@@ -16,7 +21,11 @@ const constructError = (
     if (errorMap[id]) {
       errorMapEntry = errorMap[id]
     } else if (suppliedErrorMap[id]) {
-      errorMapEntry = suppliedErrorMap[id]
+      errorMapEntry = {
+        type: `UNKNOWN`,
+        level: `ERROR`,
```

Comment:
Since both are optional for plugins (current behavior) but required internally, we're setting default values here